fluent-gecko/README.md

```diff
@@ -10,7 +10,7 @@ natural language.
 The `fluent-gecko` build system produces `*.jsm` files which are ready to be
 used as Gecko modules.
 
-`Fluent.jsm` exports the `MessageContext` constructor which provides the
+`Fluent.jsm` exports the `FluentBundle` constructor which provides the
```

Comment:
@zbraniecki Do we have a `Fluent.jsm` in `fluent-gecko`?

Reply:
No, we have MessageContext.jsm

Reply:
No, of course not. Why would we? :( Sorry, Yeah, we have MessageContext.jsm, and I'm happy to rename it to FluentBundle.jsm